common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1116,7 +1116,12 @@ def add_xml_to_node(self, node):
         node.tag = exported_node.tag
         node.text = exported_node.text
         node.tail = exported_node.tail
+
         for key, value in exported_node.items():
+            print key, value
+            if key == 'url_name' and value == 'course' and key in node.attrib:
+                # url_name for this node is already set to course run
+                continue
```

Comment:
I don't understand this comment. Should the value for `url_name` be the course run, instead of `course`?

Reply:
@pdpinch i am setting value to course run in `ExportManager`. 
This code states if the value is set in `ExportManager` then skip it setting to `course`. Otherwise set it what ever you like `course` or any key.

Reply:
update my comment

Reply:
thanks, that's much clearer.